### DISCUSSION TITLE
Triangle Tool Spawning Position

### DISCUSSION THREAD

**Discussion by cgloeckner** [xournalpp/xournalpp#6003] [Q&A]
Hey there,

I'm using xournalpp in class on a large touchscreen monitor. Most of the time, I'm zoomed in very much (making the page width fit the screen). When scrolling down (to the spot where the next page begins), I have trouble adding the triangle or compass tool.

Those tools always spawn at a static position in the upper half of the page. Scrolling or zooming to grab the tool and move it down causes some distraction to my workflow AND my student's attention.

Is there a way let those tools spawn closer to the viewport / zoomed position?

Thanks in advance.
glocke

**Comment by rolandlo** [comment]
Yes, we should change the spawing position of the geometry tools to the middle of the visible area of the current page. Currently in class I use the arrow keys to get the geometry tools to the place I want when I can't see it.

**Reply by cgloeckner** [reply]
Thanks for the quick reply.
Sure, arrow keys work, but reaching for the keyboard while standing on a touch screen isn't great either ^_^ 

Shall I create an issue (bug or feature request, your pick) for this?

**Reply by rolandlo** [reply]
Yes, please create a feature request. Check first if it doesn't exist already, because I remember having had such a discussion somewhere.

**Reply by cgloeckner** [reply]
Haven't found such an issue, so here we go: https://github.com/xournalpp/xournalpp/issues/6006
